lms/resources/_js_config/file_picker_config.py

```diff
@@ -48,10 +48,12 @@ def canvas_config(cls, request, application_instance):
         """Get Canvas files config."""
 
         files_enabled = application_instance.settings.get("canvas", "files_enabled")
+        pages_enabled = application_instance.settings.get("canvas", "pages_enabled")
 
         course_id = request.lti_params.get("custom_canvas_course_id")
         config = {
             "enabled": files_enabled,
```

Comment:
`enabled` is probably legacy naming from when "files" was the only thing that could be enabled.

We could coordinates a small PR to change this name in both the FE and BE.

Reply:
Similarly, I also noticed that the top-level property name wrapping everything else is `filePicker`, when it probably should be `contentPicker`.

Just a comment. We might want to change it, but the effort/benefit ratio is a bit discouraging 😅

Reply:
Yeah, that too.

We'd also have to make a decision about naming, probably one that we can just leave for later when we store this for course copy.  The table we have now is called "file".